src/App.tsx

```diff
@@ -1,10 +1,44 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { Footer } from './components/Footer/Footer';
+import { ErrorMessages } from './types/Errors';
+import { Errors } from './components/Errors/Errors';
+import { Header } from './components/Header/Header';
+import { Filters } from './types/Filters';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<Filters>(Filters.ALL);
+  const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
```suggestion
  const [errorMessage, setErrorMessage] = useState<ErrorMessages>(ErrorMessages.Default);
```

Add `default` option to your enum and use it here